unit_tests/test_string_original_value.py

```diff
@@ -30,7 +30,7 @@ def mutate_thrice(context):
 
 @when("Calling original_value")
 def call_original_value(context):
-    context.result = context.uut.original_value
+    context.result = helpers.str_to_bytes(context.uut.original_value())
```

Comment:
Same as before, this is what `encode()`. Doesn't that make this test redundant?

Reply:
Same as before. Testing basic functionality is useful.